content/en/service_management/app_builder/components.md

```diff
@@ -0,0 +1,767 @@
+**Provided values**: on, off
+
+### Events
+
+Event
+: **Value**: change
+
+Reaction
+: **Values**: custom, set component state, trigger query, open modal, close modal, download file
+
+For more information on events, see [Events][1].
+
+### Inspect data
+
+Displays property and value pairs in JSON format.
+{{% /collapse-content %}}
+
+
+
+{{% collapse-content title="Modal" level="h2" %}}
+Modal components have the following properties.
+
+### General
+
+Title
+: The title of the modal.<br>
+**Value**: string or expression
+
+### Appearance
+
+Size
+: The scale of the modal.<br>
+**Provided values**: sm, md, lg
+
+Is Visible
+: Determines whether the component is visible to the end-user. In edit mode, all components remain visible.<br>
+**Provided values**: on, off
+
+### Events
+
+Event
+: **Values**: toggleOpen, close, open
+
+Reaction
+: **Values**: custom, set component state, trigger query, open modal, close modal, download file
+
+For more information on events, see [Events][1].
+
+### Inspect data
+
+Displays property and value pairs in JSON format.
+
+### Example
+
+To view this component in context, see the [Metrics Explorer & Monitors Builder][2] app blueprint.
+{{% /collapse-content %}}
+
+
+
+{{% collapse-content title="Number input" level="h2" %}}
+Number input components have the following properties.
+
+Label
+: The text that displays at the top of the component.<br>
+**Value**: string or expression
+
+Default value
+: The default value that the app populates in the input box.<br>
+**Value**: number or expression that evaluates to a number
+
+Placeholder text
+: The text that displays when no value is entered.<br>
+**Value**: string or expression
+
+### Validation
+
+Min
+: The minimum value the number input accepts.<br>
+**Value**: number or expression that evaluates to a number
+
+Max
+: The maximum value the number input accepts.<br>
+**Value**: number or expression that evaluates to a number
+
+### Appearance
+
+Is Disabled
+: Applies disabled styling and removes interactions.<br>
+**Provided values**: on, off
+
+Is Visible
+: Determines whether the component is visible to the end-user. In edit mode, all components remain visible.<br>
+**Provided values**: on, off
+
+### Events
+
+Event
+: **Value**: change
+
+Reaction
+: **Values**: custom, set component state, trigger query, open modal, close modal, download file
+
+For more information on events, see [Events][1].
+
+### Inspect data
+
+Displays property and value pairs in JSON format.
+
+### Example
+
+To view this component in context, see the [ECS Task Manager][4] app blueprint.
+{{% /collapse-content %}}
+
+
+
+
+{{% collapse-content title="Radio" level="h2" %}}
+Radio components have the following properties.
+
+### General
+
+Label
+: The text that displays at the top of the component.<br>
+**Value**: string or expression
+
+Options
+: The list of radio button options that a user can select from.<br>
+**Value**: string or expression<br>
```

Comment:
I think the tooltip is incorrect for this one. It didn't accept a string for me. Same comment as above re expression